workshops/environment.yml

```diff
@@ -1,13 +1,14 @@
 name: psiml
 channels:
-  - pytorch
   - willyd
+  - pytorch
   - conda-forge
   - defaults
 dependencies:
   - blas=2.16=mkl
-  - ca-certificates=2020.6.20=hecda079_0
-  - certifi=2020.6.20=py37hc8dfbb8_0
+  - ca-certificates=2020.6.24=0
+  - certifi=2020.6.20=py37_0
+  - cpuonly=1.0=0
   - cudatoolkit=10.2.89=h74a9793_1
```

Comment:
If this works, I'm fine with leaving it in, but cudatoolkit isn't needed anymore if we're on cpu

Reply:
Noooooo, we're on GPU machines :'(